Dockerfile

```diff
@@ -3,7 +3,7 @@
 #
 FROM mcr.microsoft.com/dotnet/sdk:6.0 AS dev
 
-ENV ASPNETCORE_URLS=http://*:5000 \
+ENV ASPNETCORE_URLS=http://*:5002 \
```

Comment:
Some further context...while this is in the dev stage, it doesn't impact ops at this point.  However, we are moving toward default ports in the prod stage so that containers can be run as a non-root user.